scripts/ci/run-k8s-tests.sh

```diff
@@ -19,7 +19,8 @@ k8s_setup
 network_setup
 WITH_EBPF=true
 WITH_K8S=true
-TEST_PATTERN=K8s
+WITH_ISTIO=true
```

Comment:
I rather this be a separate test, as both k8s and istio each can exist independently (note that istio is decoupled from k8s and can be used in conjunction with docker + consul).

You also may find that k8s tests pass which when running with istio they fail ...

Reply:
Right now, we only support Istio with K8s. I don't want to have 2 jobs that spend 80% of their time setting up Minikube